Tests/FunctionalTests/Sources/FTRAlertViewTest.m

```diff
@@ -59,6 +59,11 @@ - (void)testAlertViewChain {
 }
 
 - (void)testStyledAlertView {
+  // TODO: Fix this test when the deployment target is 9.0 or greater.
+  // Link: https://github.com/google/EarlGrey/issues/645
+#if defined(__IPHONE_9_0) && (__IPHONE_OS_VERSION_MIN_REQUIRED >= __IPHONE_9_0)
```

Comment:
You could just wrap the whole test method in that #if defined?

Reply:
Done